x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/chunking/SentenceBoundaryChunkingSettingsTests.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.chunking;
+
+import org.elasticsearch.common.ValidationException;
+import org.elasticsearch.test.ESTestCase;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class SentenceBoundaryChunkingSettingsTests extends ESTestCase {
```

Comment:
`SentenceBoundaryChunkingSettings` and `WordBoundaryChunkingSettings` are serialisable objects. The test should extend `AbstractWireSerializingTestCase` instead of `ESTestCase`. `AbstractWireSerializingTestCase` will exercise the serialisation code once a couple of abstract methods are implemented.

Reply:
I'll update this test to extend `AbstractWireSerializingTestCase`. Thanks for the suggestion.